lib/files.gd

```diff
@@ -10,6 +10,11 @@
 ##
 
 
+DeclareInfoClass("InfoRead");
+if GAPInfo.CommandLineOptions.D then
+    SetInfoLevel(InfoRead, 5);
```

Comment:
Why the value 5? and not 1, or 3, or ...? Perhaps add a comment with the reason?

Reply:
Alternatively, get rid of the Info class again, as it seems atlasrep won't need/use it anyway. (But I am also fine with keeping it, just saying we have both options)

Reply:
BTW, I now wonder if we serialize info class levels when we save workspaces? If we do, then there might be a difference in behavior between checking for `GAPInfo.CommandLineOptions.D`, and checking the `InfoRead` level (i.e. when using `-D`, then saving a workspace, then loading it later on w/o `-D`;  and also vice-versa).